src/main/java/org/veupathdb/service/eda/access/repo/DB.java

```diff
@@ -14,16 +16,17 @@ interface Schema
   interface Table
   {
     String
-      Accounts          = Schema.UserAccounts + ".accounts",
-      ApprovalStatus    = Schema.StudyAccess + ".approval_status",
-      Datasets          = Schema.Tuning + ".datasetpresenter",
-      EndUsers          = Schema.StudyAccess + ".end_users",
-      EndUserHistory    = Schema.StudyAccess + ".end_user_history",
-      Providers         = Schema.StudyAccess + ".providers",
-      RestrictionLevel  = Schema.StudyAccess + ".restriction_level",
-      Staff             = Schema.StudyAccess + ".staff",
-      DatasetProperties = Schema.Tuning + ".datasetproperty",
-      UserDatasetAttributes = Schema.UserDatasets + ".datasetattributes";
+      Accounts                     = Schema.UserAccounts + ".accounts",
+      ApprovalStatus               = Schema.StudyAccess + ".approval_status",
+      Datasets                     = Schema.Tuning + ".datasetpresenter",
+      EndUsers                     = Schema.StudyAccess + ".end_users",
+      EndUserHistory               = Schema.StudyAccess + ".end_user_history",
+      Providers                    = Schema.StudyAccess + ".providers",
+      RestrictionLevel             = Schema.StudyAccess + ".restriction_level",
+      Staff                        = Schema.StudyAccess + ".staff",
+      DatasetProperties            = Schema.Tuning + ".datasetproperty",
+      AvailableUserDatasets        = "vdicontrol.availableuserdatasets",
```

Comment:
Took me a sec to realize that these hard-coded schema names are not the real names, but only used to look up SQL.  Can you add a comment above the set of these that says they're used for that and so dynamic schema names (e.g. vdi) may differ?